### PR TITLE
Finish the view.seqno renaming

### LINKED ISSUE #1187: Use consistent view.seqno terminology
We're aiming to use consistent terminology in our external API, where transactions execute in a _view_ at a _seqno_. We use this in a few places, but many places still use the old _term_ and _commit_.

- [x] Rename response headers (`x-ccf-term` to `x-ccf-view`)
- [x] Remove `x-ccf-global-commit` header (potentially a separate issue, but for clarity it would be nice to announce at the same time)
- [x] Update documentation to use new terminology. In particular, to talk about `Transaction IDs` composed of `view.seqno`
- [x] Update all external/sample code to use new terminology

### PR DESCRIPTION
**Pull request by achamayou**
fix #1187

Scrub the public APIs and RPCs to make them all aligned to the new transaction ID scheme.

### CODE REVIEW

File: src/node/history.h [hunk 1]
```diff
@@ -526,21 +526,26 @@ namespace ccf
       {
         auto version = store.next_version();
         auto view = consensus->get_view();
-        auto commit = consensus->get_commit_seqno();
+        auto commit_txid = consensus->get_committed_txid();
         LOG_DEBUG_FMT("Issuing signature at {}", version);
         LOG_DEBUG_FMT(
-          "Signed at {} view: {} commit: {}", version, view, commit);
+          "Signed at {} in view: {} commit was: {}.{}",
```
Comment (achamayou) [inline src/node/history.h:532]: Note that `view` != `commit_view`, across view changes. `view` is the view as of the signature, and is useful to the view history replication across nodes. `commit_view` is the view of the last commit at the time of that signature (`commit_seqno`).

File: src/node/rpc/common_handler_registry.h [hunk 2]
```diff
@@ -146,7 +145,7 @@ namespace ccf
         if ((nodes != nullptr) && (consensus != nullptr))
         {
           NodeId primary_id = consensus->primary();
-          auto current_term = consensus->get_view();
+          auto [current_term, _] = consensus->get_committed_txid();
```
Comment (achamayou) [inline src/node/rpc/common_handler_registry.h:148]: I'll revert this back to get_view(), I briefly thought we didn't need it anymore, but we do for signatures it turns out.

File: src/node/rpc/common_handler_registry.h [hunk 3]
```diff
@@ -264,11 +263,8 @@ namespace ccf
       };
 
       install(GeneralProcs::GET_COMMIT, json_adapter(get_commit), Read)
+        .set_execute_locally(true)
```
Comment (achamayou) [inline src/node/rpc/common_handler_registry.h:266]: That's always what we want for commit.

File: tests/infra/ccf.py [hunk 4]
```diff
@@ -596,12 +596,12 @@ def wait_for_node_commit_sync(self, consensus, timeout=3):
...
             for node in self.get_joined_nodes():
                 with node.node_client() as c:
-                    r = c.get("getCommit")
+                    r = c.get("commit")
                     commits.append(r.seqno)
             if [commits[0]] * len(commits) == commits:
                 break
             time.sleep(0.1)
-        # in pbft getCommit increments the commit version, so commits will not be the same
+        # in pbft the commit rpc increments the commit version, so commits will not be the same
```
Comment (eddyashton) [inline tests/infra/ccf.py:604]: This isn't true now that we're calling `set_execute_locally` for `commit`. If the PBFT tests work, I guess we just remove this comment?

Comment (achamayou) [reply tests/infra/ccf.py:604]: I think we can even make the check the same as for Raft.

Comment (achamayou) [reply tests/infra/ccf.py]: Yep.

**Pull request merged**